threads/test_thread_rand.c

```diff
@@ -0,0 +1,88 @@
+#include <string.h>
+#include <stdlib.h>
+#include <unistd.h>
+#include <sys/threads.h>
+
+#include "unity_fixture.h"
+
+
+typedef struct {
+	uint16_t time;
+	uint16_t id;
+} threads_self_t;
+
+
+static int cthread = 10, fin = 0;
+
+
+static char stack[10 * 512];
```

Comment:
```suggestion
static char stack[cthread * 512];
```

Reply:
As `cthread` and `stack` are now stored in struct i am not able to define size of `stack` base on `cthread` value. Switching to dynamic allocation allows for it but I want to stay simple and do not allocate and then free memory for stack.

Reply:
obsolete now.